.github/workflows/update_pardoks.yml

```diff
@@ -0,0 +1,18 @@
+name: Apply PARDOK XML Diff to DB
+
+on:
+  workflow_dispatch: {}
+  repository_dispatch:
+    types: [pardok_diff_apply]
+
+jobs:
+  pardok-xml-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Apply PARDOK XML Diff
+        run: |
+          npm ci
+          curl https://www.parlament-berlin.de/opendata/pardok-wp19.xml > pardok-wp19.xml
+          npx tsx index.ts -f pardok-wp19.xml -d postgres://${{ vars.DB_USER }}:${{ secrets.DB_PASSWORD }}@${{ vars.DB_HOST }}:${{ vars.DB_PORT }}/${{ vars.DB_NAME }} --update-db
```

Comment:
Not sure if so important since our PW is protected by beeing a secret, but `{{ vars.VALUE }}` can be logged and will not be redacted I think

Reply:
I would set the connection string  as a secret. No need to split this up